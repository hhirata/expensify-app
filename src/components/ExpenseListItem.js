import React from 'react'
import { Link } from 'react-router-dom'
const ExpenseListItem =  ({ id, description,  amount, createdAt, dispatch }) => (
    <div>
        <Link to={`/edit/${id}`}>
            <h3>Description: {description}</h3>
        </Link>
        <p>Amount: {amount}</p>
        <p>CreatedAt: {createdAt}</p>

    </div>
)

export default ExpenseListItem