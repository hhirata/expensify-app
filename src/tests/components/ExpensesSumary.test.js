import React from 'react'
import {shallow} from 'enzyme'
import {ExpensesSumary} from '../../components/ExpensesSumary'

test('should render 1 expense',()=>{
    const wrapper = shallow(<ExpensesSumary expenseCount={1} expensesTotal={10}/>)
    expect(wrapper).toMatchSnapshot()
})

test('should render 2 expenses',()=>{
    const wrapper = shallow(<ExpensesSumary expenseCount={2} expensesTotal={20}/>)
    expect(wrapper).toMatchSnapshot()
})