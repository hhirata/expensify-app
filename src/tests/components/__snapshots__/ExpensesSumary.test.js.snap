// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1 expense 1`] = `
<div>
  <h3>
    Viewing 1 expense totalling $0.10
  </h3>
</div>
`;

exports[`should render 2 expenses 1`] = `
<div>
  <h3>
    Viewing 2 expenses totalling $0.20
  </h3>
</div>
`;
